app/services/oneoffs/fix_participant_declaration_user.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Oneoffs
+  class FixParticipantDeclarationUser
+    def call
+      ActiveRecord::Base.transaction do
+        incorrect_participant_declarations.in_batches.each_record do |declaration|
+          declaration.update!(user_id: declaration.participant_profile.user.id)
```

Comment:
any chance we can use the participant profile participant identity user instead? as that's usually our source of truth? (not teacher profile user?)